exasol/slc/api/run_db_tests.py

```diff
@@ -154,6 +156,12 @@ def run_db_test(
         if external_exasol_ssh_port is None:
             raise api_errors.MissingArgumentError("external_exasol_ssh_port")
 
+    docker_runtime = None
+    if gpu_enabled:
+        additional_db_parameter += ("-enableAcceleratorDeviceDetection=1",)
+        # docker_runtime = "nvidia"
+        docker_environment_variable += ("NVIDIA_VISIBLE_DEVICES=all",)
```

Comment:
For simplicity I added the new argument `gpu_enabled` which then sets automatically the necessary arguments:
`additional_db_parameter`, `docker_runtime` and `docker_environment_variable`

Another option would be that the clients need to set all those three parameters manually